server/config.py

```diff
@@ -1,3 +1,70 @@
-import os
+import ConfigParser
+import subprocess
 
-SQLALCHEMY_DATABASE = 'postgresql://admin:1@localhost:9000/eventme'
\ No newline at end of file
+
+config = ConfigParser.RawConfigParser()
+config.read('config.cfg')
+
+"""
+    Migration
+"""
+
+alembic = config.get('alembic', 'migration')
+
+run = subprocess.Popen(alembic.split(), stdout=subprocess.PIPE)
+
+print run.communicate()
+
+"""
+    Seeding
+"""
+
+# seed_roles #
+
+seed_roles = config.get('seeding', 'seed_role')
+run = subprocess.Popen(seed_roles.split(), stdout=subprocess.PIPE)
```

Comment:
I cannot get, why you need to call as subprocess python script, why not just to import it and run a function? You don't have to fix it in scope of this PR, but it is something we need to discuss and re-work in future.